templates/404.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+ssssssssssssssssssssss
```

Comment:
że niby śpiąca strona :D? Użytkownik może tego nie załapać. Już lepiej napisać 404 not Found no i oczywiście jakoś to ostylować ładnie.

Reply:
Czyli jednak użyłem... To stronka dla mnie - bawiłem się tylko różnymi funkcjonalnościami i stąd takie kwiatki...;)